managedtenants/schemas/metadata.schema.yaml

```diff
@@ -57,6 +57,17 @@ properties:
       - AllNamespaces
       - OwnNamespace
     description: "OLM InstallMode for the addon operator."
+  syncsetMigration:
+    type: string
+    enum:
+      - "step 1 - block installations"
+      - "step 2 - orphan SSS objects"
+      - "step 3 - change SSS label"
+      - "step 4 - enable syncset"
+      - "step 5 - migration complete"
+      - "rollback ocm"
+      - "rollback selectorsyncset"
```

Comment:
I could not confirm if enum allow space in their character set. Your test pass so it should be ok.

Reply:
Thanks for raising this Sam! 
Just confirmed that the enum is being obeyed :)

Reply:
Added a test to validate false positives as well